src/server/api/analytics/stats.get.ts

```diff
@@ -1,7 +1,9 @@
 import { count, eq, not } from "drizzle-orm";
 import { jobPostingsTable, usersTable } from "~/server/db/schema";
 import authenticateAdminRequest from "~/server/utils/admin"
-
+import { NitroApp } from 'nitropack'
```

Comment:
remove this.

Also, scope of this PR is to just define logger and inject it. This PR is NOT supposed to modify actual logging. So undo those changes.

Thanks.